src/xb-main.c

```diff
@@ -21,7 +21,6 @@
 #include "xb-router.h"
 #include "xb-routed-server.h"
 
-#include <flatpak.h>
```

Comment:
Nitpick: you should be able to remove `#include <string.h>` as well, since the original commit https://github.com/endlessm/xapian-bridge/commit/91a096bc701acb8b2aff393f62a99d5913ead40b introduced it (could you just revert that commit instead? or it doesn't revert cleanly?)